reco_utils/recommender/newsrec/models/base_model.py

```diff
@@ -0,0 +1,289 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Similar to the above, can we reuse the `BaseModel` object from `deeprec`. I see the two object have similar interfaces and implementation.

Reply:
Will be include in the PR as well.

Reply:
BaseModel have several different parts from deeprec. They are changed to fit tf.keras instead of tf1.0.

Reply:
Is there particular reason why the change? It might be great to make the interface consistent across different algorithms.

Reply:
It seems tf.keras can combine with tf lower level API. I will try to merge newsrec into deeprec.

Reply:
Knowing the difference of APIs may lead to some discrepancies in evaluation metrics, so the merging may not make sense.